.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I think the entire `.vscode` directory should be added to `gitignore` since it's for your local development environment.

Reply:
Yep, that is true. But there are some settings that might have to be set for people working in this repository now? Like using whitespaces as tabs and 4whitespaces == 1 tab. Also it provieds "Debug run" mode for tests out of the box. What do you use for development? Vscode is one of the best (for me the best) .ts IDE.

Should we remove the whole folder or use maybe something like this:
```
### VisualStudioCode ###
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json

### VisualStudioCode Patch ###
# Ignore all local history of files
.history
```

I think this is the better solution right?

Reply:
added my suggested changed in -#2

Reply:
Rules for things like 4 whitespaces == 1 tab can also be set in a `.editorconfig` file, which aims to standardize these code formatting rules and editor settings across different IDEs. Then you'd set up your IDE with a plugin to support it, if it doesn't already do so (e.g. VS Code has: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)

I agree VS Code has great TS support, so it would probably be the choice editor for WC3MapTranslator.

If most of the settings you have in your .vscode environment can be replicated by EditorConfig (https://editorconfig.org/), then I'd recommend using an `.editorconfig` file. If there are things that are not supported that you'd like to be enforced in the repo, then you're free to keep the `.vscode` files.